frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -21,7 +21,6 @@ import { event as gaEvent } from "react-ga";
 import { useMenuTriggerState, useTooltipTriggerState } from "react-stately";
 import { toast } from "react-toastify";
 import styles from "./profile.module.scss";
-import BottomBannerIllustration from "../../../public/images/woman-couch-left.svg";
```

Comment:
It might be good to also delete this image from the repository - I don't expect us to remember to re-use it if design does ever decide to show the same illustration elsewhere :)